opencti-platform/opencti-front/src/components/TasksFilterValueContainer.tsx

```diff
@@ -21,6 +48,26 @@ const TasksFilterValueContainer = ({ filters }: { filters: FilterGroup }) => {
           ></TaskFilterValue>
         </React.Suspense>
       )}
+      {isFiltersInOldFormat
+        && <Chip
+          classes={{ root: classes.filter, label: classes.chipLabel }}
+          color={'warning'}
+          label={
+            <>
+              {t('deprecated format')}
+              <Tooltip
+                title={t('Filters are stored in a deprecated format (before 5.12)')}
+              >
+                <InformationOutline
+                  fontSize="small"
+                  color="secondary"
+                  style={{ cursor: 'default' }}
+                />
+              </Tooltip>
+            </>
+          }
+        />
+      }
```

Comment:
On a second thought, we might as well remove this as we already convert it on the fly to new format. This information has no effect on the user, he does not care and won't do anything about it.

what do you think @Archidoit ?

Reply:
ok, I do it